src/js/player.js

```diff
@@ -2464,23 +2551,29 @@ class Player extends Component {
     }
 
     // intial sources
-    this.cache_.sources = sources;
     this.changingSrc_ = true;
 
-    // intial source
-    this.cache_.source = sources[0];
```

Comment:
this is still not being set unless `this.updateSourceCaches_` sets it?

Reply:
`updateSourceCaches_` will set it.

Reply:
https://github.com/videojs/video.js/pull/5040/files#diff-3b0266ff1c037b289ec624ab25b0272eR1224